Commandant/ArgumentParser.swift

```diff
@@ -61,15 +61,19 @@ public final class ArgumentParser {
 	/// Initializes the generator from a simple list of command-line arguments.
 	public init(_ arguments: [String]) {
 		// The first instance of `--` terminates the option list.
-		let params = split(arguments, maxSplit: 1, allowEmptySlices: true) { $0 == "--" }
+		let params = arguments.split(1, allowEmptySlices: true) { $0 == "--" }
```

Comment:
They must've changed it back because I didn't see any failures locally when running the tests.